bodhi/server/util.py

```diff
@@ -601,7 +601,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
```

Comment:
ooooh - I had mistakenly thought this line was logging out at info rather than debug. My apologies for the confusion on the test. Let's use the test to assert that the stdout is getting logged as debug.